test/integration/src/test/java/io/pravega/test/integration/ClusterWrapperTest.java

```diff
@@ -94,4 +127,81 @@ public void writeAndReadBackAMessageWithTlsAndAuthEnabledServer() {
+
+        // Setup and start the cluster
+        @Cleanup
+        ClusterWrapper cluster = ClusterWrapper.builder()
+                .controllerRestEnabled(true)
+
+                // Auth related configs
+                .authEnabled(true)
+                .passwordAuthHandlerEntries(TestUtils.preparePasswordInputFileEntries(passwordInputFileEntries, restApiUserPwd))
+
+                // TLS related configs
+                .tlsEnabled(true)
+                .tlsServerCertificatePath(TestUtils.pathToConfig() + SecurityConfigDefaults.TLS_SERVER_CERT_FILE_NAME)
+                .tlsServerKeyPath(TestUtils.pathToConfig() + SecurityConfigDefaults.TLS_SERVER_PRIVATE_KEY_FILE_NAME)
+                .tlsHostVerificationEnabled(false)
+                .tlsServerKeystorePath(TestUtils.pathToConfig() + SecurityConfigDefaults.TLS_SERVER_KEYSTORE_NAME)
+                .tlsServerKeystorePasswordPath(TestUtils.pathToConfig() + SecurityConfigDefaults.TLS_PASSWORD_FILE_NAME)
+
+                .build();
+        cluster.start();
+
+        // Setup REST client config
+        org.glassfish.jersey.client.ClientConfig clientConfig = new org.glassfish.jersey.client.ClientConfig();
+        clientConfig.register(JacksonJsonProvider.class);
+        clientConfig.property("sun.net.http.allowRestrictedHeaders", "true");
+        clientConfig.register(HttpAuthenticationFeature.basic(restApiUser, restApiUserPwd));
+
+        // Prepare a TLS context with truststore containing the signing CA's vertificate
+        KeyStore trustStore = createTrustStore(TestUtils.pathToConfig() + SecurityConfigDefaults.TLS_CA_CERT_FILE_NAME);
+        TrustManagerFactory tmf = TrustManagerFactory.getInstance(TrustManagerFactory.getDefaultAlgorithm());
+        tmf.init(trustStore);
+        SSLContext tlsContext = SSLContext.getInstance("TLS");
+        tlsContext.init(null, tmf.getTrustManagers(), null);
+
+        // Instantiate the REST client
+        javax.ws.rs.client.Client client = ClientBuilder.newBuilder()
+                        .withConfig(clientConfig)
+                        .sslContext(tlsContext)
+                        .build();
+
+        String restServerURI = cluster.controllerRestUri();
+        log.info("REST Server URI: {}", restServerURI);
+
+        // Invoke the REST operation
+        String resourceURl = new StringBuilder(restServerURI).append("/ping").toString();
+        WebTarget webTarget = client.target(resourceURl);
+        Invocation.Builder builder = webTarget.request();
+
+        // Check if the response was as expected
+        Response response = builder.get();
+        assertEquals("Response to /ping was not OK", OK.getStatusCode(), response.getStatus());
+        log.info("Ping successful.");
+    }
+
+    //region Unit test
+    @Test
+    public void setsDefaultValuesWhenBuilderSpecifiesNoValues() {
+        ClusterWrapper objectUnderTest = ClusterWrapper.builder().build();
```

Comment:
You should clean this up

Reply:
Ok. I've removed the unit tests from this integration test class in the latest commit. Have pushed the assertions for checking default values into an integration test above.